packages/playwright-core/src/cli/program.ts

```diff
@@ -608,8 +614,10 @@ async function launchContext(options: Options, extraOptions: LaunchOptions): Pro
   return { browser, browserName: browserType.name(), context, contextOptions, launchOptions };
 }
 
-async function openPage(context: BrowserContext, url: string | undefined): Promise<Page> {
-  const page = await context.newPage();
+async function openPage(context: BrowserContext, url: string | undefined, allowPageReuse: boolean = false): Promise<Page> {
```

Comment:
Now that we support `--user-data-dir` option in most commands, it looks like all the callers of `openPage` actually want to `allowPageReuse`. I think we can safely remove this argument.